tests/util/test_unit_conversion.py

```diff
@@ -345,30 +345,30 @@ def test_power_convert(
     [
         (1000, PRESSURE_HPA, pytest.approx(14.5037743897), PRESSURE_PSI),
         (1000, PRESSURE_HPA, pytest.approx(29.5299801647), PRESSURE_INHG),
-        (1000, PRESSURE_HPA, pytest.approx(100000), PRESSURE_PA),
-        (1000, PRESSURE_HPA, pytest.approx(100), PRESSURE_KPA),
-        (1000, PRESSURE_HPA, pytest.approx(1000), PRESSURE_MBAR),
-        (1000, PRESSURE_HPA, pytest.approx(100), PRESSURE_CBAR),
+        (1000, PRESSURE_HPA, 100000, PRESSURE_PA),
+        (1000, PRESSURE_HPA, 100, PRESSURE_KPA),
+        (1000, PRESSURE_HPA, 1000, PRESSURE_MBAR),
+        (1000, PRESSURE_HPA, 100, PRESSURE_CBAR),
```

Comment:
Please do not drop the `pytest.approx`, there's no reason to test for equality of floating point numbers here